cssFiles/cardfiles/cardswithbadges.css

```diff
@@ -0,0 +1,111 @@
+.container1{
+    position: relative;
+    border: 1rem;
+    margin: 1rem;
+    width: 250px;
+    border-radius: 10px;
+    
+  }
+  
+  .ecom-img1{
+    position: relative;
+    height: 15rem;
+    width: 100%;
+    border-radius: 5px 5px 5px 5px;
+    
+  }
+  
+  .secondary-container1{
+    display: flex;
+    /* gap: 15px; */
+    
+  }
+  
+  .actual-price1 {
+      display: inline;
+      margin-right: 4px;
+      color: grey;
+      text-decoration: line-through;
+    }
+    .disc-price1 {
+      margin-right: 4px;
+      display: inline;
+    }
+    .offer-info1 {
+      margin-right: 4px;
+      display: inline;
+      color: darkgreen;
+    }
+  .card-footer1 {
+      display: flex;
+      flex-wrap: wrap;
+      align-items: center;
+      margin-top: auto;
+      justify-content: center;
+      gap: 1rem;
+    }
+    .btn-link1 {
+      background: transparent;
+      border: 0;
+      font-size: 1.2rem;
+      padding: 5px 10px;
+      border-radius: 5px;
+    }
+    .call-to-action1 {
+      margin-top: 1rem;
+      background-color: darkorange;
+      color: var(--white);
+    }
+    .call-to-action1:hover {
+      background-color: orange;
+    }
+  
+    .vertical-btn1{
+      width: 100%;
+    }
+  
+     .card-badge1 {
+      z-index: 2;
+      position: absolute;
+      left: .025rem;
+      background: orange;
+      color: var(--lynx-white);
+      padding: 5px;
+      font-size:smaller
+    }
+  
+     .heart-badge1 {
+      z-index: 2;
+      position: absolute;
+      right: .03rem;
+      color:red;
+      padding: 5px;
+      
+    }
+
+    .cardnew{
+        min-height: 100px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        gap: .5rem;
+        margin: auto;
+        box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+        border-radius: 4px;
+        background-color: #F5F7F9;
+        width: 100%;
+        padding: 1rem;
+        margin-bottom: 1.5rem;
+        flex-wrap: wrap;
+    
+    }
+
+
+h6{
```

Comment:
Instead of targetting the `tag selectors` directly you can try giving them a `class`.